Source/UnitTests/Core/DSP/DSPTestText.cpp

```diff
@@ -1,4 +1,10 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
 
+#include "DSPTestText.h"
+
+const char* s_dsp_test_text = R"(
```

Comment:
Does `extern const char s_dsp_test_text[]` (and accordingly for the source) solve the problem? It compiles on my computer.